apps/chat-e2e/src/testData/prompts/promptData.ts

```diff
@@ -51,7 +51,8 @@ export class PromptData extends FolderData {
   ): FolderPrompt {
     const prompt = this.preparePrompt(content, description);
     const folder = this.prepareFolder();
-    prompt.folderId = folder.id;
+    prompt.folderId = folder.folderId;
+    prompt.id = `${folder.folderId}/${prompt.id}`;
```

Comment:
if you want to put prompt inside this folder then it should be like that
```suggestion
    prompt.folderId = folder.id;
    prompt.id = `${folder.id}/${prompt.id}`;
```
because in current code you put a prompt inside parent folder

Reply:
in this method folder object looks like this:
`{
  "id": "5JYIzSEgZ9",
  "name": "uQGHULu",
  "type": "prompt",
  "folderId": "uQGHULu"
}`

since folder name is used to build the path to prompt/conversation, I set:
`prompt.folderId = folder.id`
or equals 
`prompt.folderId = folder.name`

In the next step, where prompt is created, bucket is added to the prompt path